slycot/math.py

```diff
@@ -137,19 +138,21 @@ def mb05md(a, delta, balanc='N'):
     arg_list = [ 'balanc', 'n', 'delta', 'a', 'lda'+hidden, 'v', 'ldv'+hidden,
                  'y','ldy'+hidden,'valr','vali',
                  'iwork'+hidden,'dwork'+hidden,'ldwork'+hidden,'info'+hidden]
-    out = _wrapper.mb05md(balanc=balanc,n=min(a.shape),delta=delta,a=a)
-    if out[-1] == 0:
-        return out[:-1]
-    elif out[-1] < 0:
-        error_text = "The following argument had an illegal value: "+arg_list[-out[-1]-1]
-    elif out[-1] > 0 and out[-1] <= n:
-        error_text = "Incomplete eigenvalue calculation, missing %i eigenvalues" % out[-1]
-    elif out[-1] == n+1:
+    n=min(a.shape)
+    (Ar,Vr,Yr,VALr,VALi,INFO) = _wrapper.mb05md(balanc=balanc,n=n,delta=delta,a=a)    
+    if INFO == 0:
+        VAL=VALr+1J*VALi
+        return (Ar,Vr,Yr,VAL)
```

Comment:
The docstring headline is `Ar, Vr, Yr, VALRr, VALDr = mb05md(a, delta, balanc='N')` - would you mind checking if that's wrong (it doesn't match this return), and, if so, correct it too?

Reply:
Yes it was wrong. Before it returned `VALr` and `VALi` as real and imaginary part of the eigenvalues. My change returns the complex VAL. (With a check added in commit af8927f if it is necessary. Returns real list otherwise.)

Docstring fixed in 961f172.